modules/core/src/debug/index.js

```diff
@@ -0,0 +1,53 @@
+const log = require('../utils/log').default;
+let enabled = false;
+let loggers;
+// Conditionally load default loggers in development mode
+// eslint-disable-next-line
+if (process.env.NODE_ENV !== 'production') {
+  loggers = require('./loggers').getLoggers(log);
+}
+
+function enable(state = true) {
+  enabled = state;
+  if (state) {
+    log.enable();
+  }
+  if (state && !loggers) {
+    // TODO - load from a pre-bundled URL
+    loggers = {};
+  }
+}
+
+function register(getLoggers) {
+  loggers = getLoggers(log);
+}
+
+function logEvent(eventType, source, arg1, arg2) {
+  if (enabled && loggers[eventType]) {
+    // Not using ...args to defeat perf hit from array construction
+    loggers[eventType](source, arg1, arg2);
+  }
+}
+
+module.exports = {
+  init: deck => {
```

Comment:
Nit: `init` is a bit long for being an inline function? Move out and just reference it here?